libraries/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -49,13 +52,33 @@ export class InstallAction extends BaseInstallAction {
       parameterLongName: '--check-only',
       description: `Only check the validity of the shrinkwrap file without performing an install.`
     });
+
+    this._ignoreScriptsParameter = this.defineFlagParameter({
+      parameterLongName: '--ignore-scripts',
+      description:
+        'Do not execute any install lifecycle scripts specified in package.json files and its' +
+        ' dependencies when "rush install". Running with this flag makes your installing faster. Later,' +
+        ' you can run "rush install" to run all ignored scripts. Moreover, you can partial install such as' +
+        ' "rush install --to <package>" to run ignored scripts of the dependencies of the selected projects.'
+    });
   }
 
   protected async buildInstallOptionsAsync(): Promise<IInstallManagerOptions> {
     const terminal: Terminal = new Terminal(new ConsoleTerminalProvider());
+
+    const pnpmFilterArguments: string[] = await this._selectionParameters!.getPnpmFilterArgumentsAsync(
+      terminal
+    );
+
+    let selectedProjects: Set<RushConfigurationProject> | undefined;
+    if (pnpmFilterArguments.length > 0) {
+      selectedProjects = await this._selectionParameters!.getSelectedProjectsAsync(terminal);
+    }
```

Comment:
If you need both of these at once going forwards it makes sense to update the `getPnpmFilterArgumentsAsync` API to also return the selectedProjects to avoid duplicating work.